package.json

```diff
@@ -40,14 +40,14 @@
         "@types/node": "13.7.4",
         "@typescript-eslint/eslint-plugin": "2.20.0",
         "@typescript-eslint/parser": "2.20.0",
-        "auto": "9.13.0",
+        "auto": "^9.19.5",
         "eslint": "6.8.0",
         "eslint-config-prettier": "6.10.0",
         "eslint-loader": "3.0.3",
         "eslint-plugin-import": "2.20.1",
         "eslint-plugin-jest": "23.8.0",
         "eslint-plugin-prettier": "3.1.2",
-        "jest": "25.1.0",
+        "jest": "^25.1.0",
```

Comment:
don't forget to rerun yarn, you have changes

Reply:
@ardeois already done it )